### PR TITLE
M9

### PR DESCRIPTION
**Pull request by Lucas-Fernandes-Nogueira**

### CODE REVIEW

**Comment by Lucas-Fernandes-Nogueira** [comment]
I've done all the required parts and it all seems to be working.

File: .env.template
Comment (JackMeadDev) [inline .env.template:8]: Well remembered to update these

File: .travis.yml
Comment (JackMeadDev) [inline .travis.yml]: Not important, but you can save some space by passing these environment variables through in the short form, e.g.
`docker run -e MONGO_USERNAME -e MONGO_URL ... ` which will default to use the environment variable of the same name

File: app.py
Comment (JackMeadDev) [inline app.py]: Minor, but here's a good example of where we could prevent implementation details leaking - just making this variable name a little more generic (`item_storage`?) would save this class from having to update as many places in future, and we could essentially treat our storage solution as an interface that we only care about the specifics of on line 8, which you've demonstrated by keeping the same method names on MongoApi

File: mongo_api.py
Comment (JackMeadDev) [inline mongo_api.py]: We've added this but don't use it (since the client will do it by default) so might as well just remove this?

File: tests/integration-tests/test_app.py
Comment (JackMeadDev) [inline tests/integration-tests/test_app.py]: Leftover print statement :)

**Review by JackMeadDev — APPROVED** [review]
Nice work, this looks really neat and all just works - good job clearing out all the old references to trello. I've left a couple of minor thoughts below, but happy for you to merge this when you're ready.

Your heroku app is currently down - I'm treating that as a separate issue as I can see that this branch hasn't deployed to heroku, but we should investigate that if it persists after merging this.

File: app.py
Comment (Lucas-Fernandes-Nogueira) [reply app.py]: Thanks!

File: .travis.yml
Comment (Lucas-Fernandes-Nogueira) [reply .travis.yml]: Done 👍!

File: mongo_api.py
Comment (Lucas-Fernandes-Nogueira) [reply mongo_api.py]: Agree!

File: tests/integration-tests/test_app.py
Comment (Lucas-Fernandes-Nogueira) [reply tests/integration-tests/test_app.py]: Embarrassing 😅.

**Pull request merged**
